File: web-app/src/app/norm/entity/student.ts
import {Klass} from './klass';

export class Student {
  id: number;
  klass: Klass;
  name: string;
  sno: string;


  constructor(data?: { id?: number; name?: string; sno?: string; klass?: Klass }) {
    if (!data) {
      return;
    }
    this.id = data.id ? data.id : null;
    this.klass = data.klass ? data.klass : null;
    this.name = data.name ? data.name : '';
    this.sno = data.sno ? data.sno : '';

  }
}

File: api/src/main/java/com/huawei/Dao/model/Student.java
package com.huawei.Dao.model;

import java.io.Serializable;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Getter;
import lombok.Setter;

/**
 * student
 * @author 
 */
@Getter
@Setter
public class Student{
    private String id;

    @JsonProperty("sno")
    private String studentId;

    private Long klassId;

    private String name;
}